### PR TITLE
Result detail

### PR DESCRIPTION
**Pull request by d-opri**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
   return (
     <>
       <Activity />
+      {bill && (
+        <BillDetails title={bill.title} total={bill.amount}>
+          {bill.participants.map((participant) => {
+            <div key={participant.id}>
+              {participant.name}
+              {participant.balance}
+            </div>;
```
Comment (Olga-Mishareva) [inline pages/index.js:34]: It's looks like a list with items :)

File: pages/create-bill/index.js [hunk 2]
```diff
@@ -147,18 +121,29 @@ export default function BillForm() {
...
-            </div>
-          ))}
-        </BillDetails>
+        <>
+          <BillDetails title={title} total={amount}>
+            {results.map((participant, index) => (
+              <div key={index}>
+                {participant.name}
+                {participant.balance}
+              </div>
```
Comment (Olga-Mishareva) [inline pages/create-bill/index.js:130]: It's looks like a list with items :)

**Review by Olga-Mishareva — COMMENTED** [review]
Hi Danie,
it looks good! Only one point. I guess, it should be a list by mapping :)

File: pages/index.js (hunk 1 above)
Comment (d-opri) [reply pages/index.js:34]: youre right! thanks for pointing it out

**Pull request merged**